gencertbuf.pl

```diff
@@ -210,6 +222,23 @@
 
 print OUT_FILE "#endif /* HAVE_PQC */\n\n";
 
+# convert and print dilithium keys
+print OUT_FILE "#ifdef HAVE_PQC\n\n";
```

Comment:
Should this also be wrapped in `HAVE_DILITHIUM`? Might consider the same for Falcon above.